cmd/statusd/main.go

```diff
@@ -19,20 +19,34 @@ var (
 )
 
 var (
-	prodMode       = flag.Bool("production", false, "Whether production settings should be loaded")
-	nodeKeyFile    = flag.String("nodekey", "", "P2P node key file (private key)")
-	dataDir        = flag.String("datadir", params.DataDir, "Data directory for the databases and keystore")
-	networkID      = flag.Int("networkid", params.RopstenNetworkID, "Network identifier (integer, 1=Homestead, 3=Ropsten, 4=Rinkeby, 777=StatusChain)")
-	whisperEnabled = flag.Bool("shh", false, "SHH protocol enabled")
-	swarmEnabled   = flag.Bool("swarm", false, "Swarm protocol enabled")
-	httpEnabled    = flag.Bool("http", false, "HTTP RPC endpoint enabled (default: false)")
-	httpPort       = flag.Int("httpport", params.HTTPPort, "HTTP RPC server's listening port")
-	ipcEnabled     = flag.Bool("ipc", false, "IPC RPC endpoint enabled")
-	cliEnabled     = flag.Bool("cli", false, "Enable debugging CLI server")
-	cliPort        = flag.String("cliport", debug.CLIPort, "CLI server's listening port")
-	logLevel       = flag.String("log", "INFO", `Log level, one of: "ERROR", "WARN", "INFO", "DEBUG", and "TRACE"`)
-	logFile        = flag.String("logfile", "", "Path to the log file")
-	version        = flag.Bool("version", false, "Print version")
+	prodMode        = flag.Bool("production", false, "Whether production settings should be loaded")
+	nodeKeyFile     = flag.String("nodekey", "", "P2P node key file (private key)")
+	dataDir         = flag.String("datadir", params.DataDir, "Data directory for the databases and keystore")
+	networkID       = flag.Int("networkid", params.RopstenNetworkID, "Network identifier (integer, 1=Homestead, 3=Ropsten, 4=Rinkeby, 777=StatusChain)")
+	whisperEnabled  = flag.Bool("shh", false, "SHH protocol enabled")
+	swarmEnabled    = flag.Bool("swarm", false, "Swarm protocol enabled")
+	httpEnabled     = flag.Bool("http", false, "HTTP RPC endpoint enabled (default: false)")
+	httpPort        = flag.Int("httpport", params.HTTPPort, "HTTP RPC server's listening port")
+	ipcEnabled      = flag.Bool("ipc", false, "IPC RPC endpoint enabled")
+	cliEnabled      = flag.Bool("cli", false, "Enable debugging CLI server")
+	cliPort         = flag.String("cliport", debug.CLIPort, "CLI server's listening port")
+	logLevel        = flag.String("log", "INFO", `Log level, one of: "ERROR", "WARN", "INFO", "DEBUG", and "TRACE"`)
+	logFile         = flag.String("logfile", "", "Path to the log file")
+	version         = flag.Bool("version", false, "Print version")
+	printWhisperCfg = flag.Bool("printwhispcfg", false, "Print whisper config")
```

Comment:
Generally, we want to keep an amount of command line flags as low as possible, and this flag doesn't seem very useful. I think it's safe to remove printing config at all.

Reply:
removing printwhispcfg and corresponding function that prints it